src/FetchWorkerTool.ts

```diff
@@ -33,22 +45,26 @@ class PrivateFetchWorkerTool {
 
         try {
             if (this.isGetSupported) {
-                // eslint-disable-next-line global-require
-                const FetchWorker = require('worker-loader?{"inline":true,"fallback":true}!./FetchWorkerTool.worker');
-
-                const worker = new FetchWorker();
+                // Yes, this is a browser API and we've specified `browser: false` in the eslint env,
+                // but `isGetSupported` checks for the presence of Worker and uses it only if present.
+                // Also see https://webpack.js.org/guides/web-workers/
+                // eslint-disable-next-line no-undef
+                const worker = new Worker(
+                    /* webpackChunkName: "fetch-worker" */ new URL('./FetchWorkerTool.worker', import.meta.url)
+                );
 
                 worker.addEventListener('message', ({data}) => {
                     if (data.support) {
                         this._workerSupport = data.support;
                         return;
                     }
                     for (const message of data) {
-                        if (this.jobs[message.id]) {
+                        const job = this.jobs[message.id];
```

Comment:
😎

Reply:
Had to please TS here since it's an access and it doesn't know that it's non-null later 😄